README.md

```diff
@@ -158,6 +158,26 @@ plugin "nomad-driver-podman" {
 }
 ```
 
+* socket stanza: If you require to use multiple podman sockets (rootful/rootless), you can define a list of them. Mutually exclusive with socket_path option.
```

Comment:
HCL has "blocks", not "stanzas" (lots of docs have this wrong, unfortunately). Also, because the `socket` block itself has a `socket_path`, it'd be good to disambiguate which field we mean just to be super-clear. How about:

```suggestion
* socket block: Configures a single podman socket. You can define multiple `socket` blocks if you need to use multiple podman sockets (for example, rootless vs rootful sockets). Mutually exclusive with the top-level `plugin.config.socket_path` option.
```